Add screenshots to English documentation (#120)
wayfair-contribs/plentymarkets-plugin: meta/documents/user_guide/en/plugin_installation.md

1. Log in to the Plentymarkets system.

> 2. From the main Plentymarkets page, go to `plugins` >> `plentyMarketplace`, which will use the Plentymarkets system's owner’s credentials to log you in on the plentyMarketplace website:

Reviewer: ```suggestion
2. From the main Plentymarkets page, go to `Plugins` >> `plentyMarketplace`
```

7. Click on the `Wayfair` entry in the list. If you have previously installed the Wayfair plugin via the "git" channel, there may multiple entries for "Wayfair." Please select the entry that has the `Marketplace` source (![marketplace icon](../../../images/common/icon_marketplace.png)).

8. Review the information on the page:

    ![wayfair plugin info](../../../images/en/installation/marketplace_wayfair.png)

9. In the `Select version` drop-down menu, choose [the newest plugin version](https://github.com/wayfair-contribs/plentymarkets-plugin/releases) if it is not already selected.

10. Click the `Install` button ![install button](../../../images/en/installation/button_install.png) at the top of the page. You will be redirected to the individual Plugin Set's details.

11. Verify that a row exists on the page with the `Name` displayed as "Wayfair" and the `Installed` version showing as the desired version

12. In the `Active` column for the "Wayfair" row, click on the switch so that it moves to the right-side, enabled position. **Failure to activate the Wayfair plugin will prevent its functionalities from being available**.

> 13. Verify that the Wayfair plugin's row looks like this:

Reviewer: ```suggestion
13. Verify that the Wayfair plugin's row looks the same as the screenshot below (your `Installed` version may be different)
```

    ![wayfair plugin not yet deployed](../../../images/en/installation/wayfair_plugin_not_yet_deployed.png)

14. Click the `Deploy plugin set` button (appears as a Save button):

    ![deploy plugin set button](../../../images/en/installation/button_deploy_plugin_set.png)

15. A progress bar will appear while the plugin is installed.

> 16. Confirm that the `Deployed` column for the "Wayfair" row is now populated, and it reflects the value in the `Installed` column:

Reviewer: ```suggestion
16. Confirm that the `Deployed` column for the "Wayfair" row is now populated, and it reflects the value in the `Installed` column (your `Installed` or `Deployed` version may be different)
```
